packages/logger/src/middleware/middy.ts

```diff
@@ -1,6 +1,13 @@
 import type { Logger } from '../Logger';
 import middy from '@middy/core';
 
+/**
+ * Middy middleware that adds the current Lambda invocation's context
+ * inside all log items.
```

Comment:
an example can be useful here , and can will be more in line with what we have in tracer for instance: https://github.com/awslabs/aws-lambda-powertools-typescript/blob/main/packages/tracing/src/middleware/middy.ts#L15

Reply:
I agree, well spotted!